sdk/resourcemanager/Azure.ResourceManager.Core/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+-Initial checkin
```

Comment:
Here is a link to our changelog guidelines http://aka.ms/azsdk/guideline/changelogs. You will probably want to add some of the well-known heading so things get picked up automatically for our release note processes.